init.lua

```diff
@@ -376,16 +378,23 @@ local function lootCorpse(corpseID)
             if corpseItem() then
                 local stackable = corpseItem.Stackable()
                 local freeStack = corpseItem.FreeStack()
-                if corpseItem.NoDrop() then
-                    table.insert(noDropItems, corpseItem.ItemLink('CLICKABLE')())
-                elseif corpseItem.Lore() then
+                if corpseItem.Lore() then
                     local haveItem = mq.TLO.FindItem(('=%s'):format(corpseItem.Name()))()
                     local haveItemBank = mq.TLO.FindItemBank(('=%s'):format(corpseItem.Name()))()
                     if haveItem or haveItemBank or freeSpace <= loot.SaveBagSlots then
                         table.insert(loreItems, corpseItem.ItemLink('CLICKABLE')())
+                    elseif corpseItem.NoDrop() then
+                            if loot.LootNoDrop then 
+                                lootItem(i, getRule(corpseItem), 'leftmouseup')
+                            else
+                                table.insert(noDropItems, corpseItem.ItemLink('CLICKABLE')())
+                            end
                     else
                         lootItem(i, getRule(corpseItem), 'leftmouseup')
                     end
+                elseif corpseItem.NoDrop() then
+                        if loot.LootNoDrop then lootItem(i, getRule(corpseItem), 'leftmouseup') end
```

Comment:
Can keep the else condition to track the no drop item that is being left behind since it was removed above.